tool/src/org/antlr/v4/Tool.java

```diff
@@ -139,10 +139,6 @@ public Option(String fieldName, String name, OptionArgType argType, String descr
 	protected boolean haveOutputDir = false;
 	protected boolean return_dont_exit = false;
 
-    // The internal options are for my use on the command line during dev
-    public static boolean internalOption_PrintGrammarTree = false;
```

Comment:
i think i use this don't I (might leave in for debuggin)

Reply:
I tried to minimize number of static fields since they could cause multithreading issues, it doesn't look safe. Also, it looks like temporary code that is used only during testing. I suggest either using temporary code without comitting or creating permanent option without static field.